### PR TITLE
2020 csv alignment

### LINKED ISSUE #2020: CSVEntry: Icon and number spacing is off
- [ ] Fix alignment issue with numbers/rows/icons
- [ ] Fix extra space added at bottom when adding rows

### PR DESCRIPTION
**Pull request by benjamincharity**
- Fix height alignment with the external columns
- Reverse order of lint commands to hopefully not squash output)

Closes #2020

### CODE REVIEW

File: terminus-ui/csv-entry/src/csv-entry.component.ts [hunk 1]
```diff
@@ -131,6 +131,11 @@ export class TsCSVEntryComponent implements OnInit, OnDestroy {
    */
   protected uid = `ts-csv-entry-${nextUniqueId++}`;
 
+  /**
+   * Define the static height needed in the DOM for the external rows
+   */
+  public fakeRowHeight = '35px';
```
Comment (atlwendy) [inline terminus-ui/csv-entry/src/csv-entry.component.ts:137]: Magic number 😄

File: terminus-ui/csv-entry/src/csv-entry.component.html [hunk 2]
```diff
@@ -94,7 +94,7 @@
       <!-- Column for delete buttons -->
       <div fxLayout="column">
         <!-- Fake row for header -->
-        <div style="height: 47px" fxFlex="noshrink"></div>
+        <div [style.height]="fakeRowHeight"  fxFlex="0 0 auto"></div>
```
Comment (atlwendy) [inline terminus-ui/csv-entry/src/csv-entry.component.html:97]: cool... didn't know you could do this....

**Review by atlwendy — APPROVED** [review]
Thought 2020 in the title was for the year... lol

File: terminus-ui/csv-entry/src/csv-entry.component.ts (hunk 1 above)
Comment (benjamincharity) [reply terminus-ui/csv-entry/src/csv-entry.component.ts:137]: yeah.. unfortunately, at least for now, it truly is magic 😂

File: terminus-ui/csv-entry/src/csv-entry.component.html (hunk 2 above)
Comment (benjamincharity) [reply terminus-ui/csv-entry/src/csv-entry.component.html:97]: Yeah, it somehow felt a _little_ less dirty than inlining the style directly..

**Pull request merged**